components/Card/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import styled from "styled-components";
+
+interface Props extends React.Attributes {
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+  onClick?: () => void;
+  hoverable?: boolean;
```

Comment:
Sort them in React's standard props order such like: others -> event handlers (`onXxx`) -> `className` -> `style` -> `children`

```suggestion
  hoverable?: boolean;
  onClick?: () => void;
  className?: string;
  style?: React.CSSProperties;
  children?: React.ReactNode;
```

Reply:
How do I know React's standard props order?

Reply:
What I've done is looking at bunch of components in the world, components React officially provides, and from several famous UI component libraries.